geniza/corpus/admin.py

```diff
@@ -207,47 +215,45 @@ def tabulate_queryset(self, queryset):
             all_textblocks = doc.textblock_set.all()
             all_footnotes = doc.footnotes.all()
 
-            row = {
+            initial_entry = doc.log_entries.first()
+            latest_revision = doc.log_entries.last()
+
+            row = self.DocumentRow(**{
```

Comment:
@kmcelwee wanted to preserve a mapping between field names and values, but without duplicating the list of valid field names. `export_to_csv` needs the full list of field names to write the column headers, but since they were only defined in a literal in `tabulate_queryset` you would otherwise need to duplicate them. the namedtuple was my suggestion to define the list of valid fields once so that it's accessible in both `tabulate_queryset` and `export_to_csv`, as well as throwing an error if you try to assign a field that doesn't exist.